pkg/controllers/managementuser/rkecontrolplanecondition/rkecontrolplanecondition.go

```diff
@@ -16,38 +16,40 @@ import (
 	upgradev1 "github.com/rancher/rancher/pkg/generated/controllers/upgrade.cattle.io/v1"
 	"github.com/rancher/rancher/pkg/namespace"
 	"github.com/rancher/rancher/pkg/settings"
-	"github.com/rancher/rancher/pkg/types/config"
 	"github.com/sirupsen/logrus"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type handler struct {
-	mgmtClusterName      string
-	clusterCache         provisioningcontrollers.ClusterCache
-	downstreamAppClient  catalogv1.AppClient
-	downstreamPlanClient upgradev1.PlanClient
+	MgmtClusterName      string
```

Comment:
Field names should be unexported again as well

Reply:
done!